### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by JacintaNgwu**
- What animals belong to Melody Pond?
image

- List of all animals that are pokemon (their type is Pokemon).
image

- List all owners and their animals, remember to include those that don't own any animal.
image

- How many animals are there per species?
image

- List all Digimon owned by Jennifer Orwell.
image

- List all animals owned by Dean Winchester that haven't tried to escape.
image

- Who owns the most animals?
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -49,4 +49,33 @@ GROUP BY species;
 
 SELECT species, AVG(escape_attempts) FROM animals
 WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+SELECT name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
```
Comment (Gambit142) [inline queries.sql:56]: - [x] One of the requirements is that you include the snapshot of queries in your pull request summary. The snapshot for the result of this query is incorrect. Kindly include the correct snapshot. 👍

File: queries.sql [hunk 2]
```diff
@@ -49,4 +49,33 @@ GROUP BY species;
...
+
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
```
Comment (Gambit142) [inline queries.sql:60]: - [x] One of the requirements is that you include the snapshot of queries in your pull request summary. The snapshot for the result of this query is incorrect. Kindly include the correct snapshot. 👍

File: queries.sql [hunk 3]
```diff
@@ -49,4 +49,33 @@ GROUP BY species;
...
+
+SELECT owners.full_name, animals.name FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
```
Comment (Gambit142) [inline queries.sql:63]: - [x] One of the requirements is that you include the snapshot of queries in your pull request summary. The snapshot for the result of this query is incorrect. Kindly include the correct snapshot. 👍

File: queries.sql [hunk 4]
```diff
@@ -49,4 +49,33 @@ GROUP BY species;
...
+
+SELECT species.name, COUNT(animals.id) as count FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.id;
+
```
Comment (Gambit142) [inline queries.sql]: - [x] The query gives the wrong result. The correct query should bring the total number of _**Digimons**_ and the total number of _**Pokemons**_. Kindly write the correct query and update the screenshot. 👍 

Hint: When creating the `animals` table, that `ID` was not serial, so your current `animals` table has no data for ID. So, I would advise you to either make the `ID` serial or change `count(animals.id)` to `count(animals.species_id)`. 👍 😎

File: queries.sql [hunk 5]
```diff
@@ -49,4 +49,33 @@ GROUP BY species;
...
+SELECT animals.name FROM animals
+JOIN species ON animals.species_id = species.id
+JOIN owners ON animals.owner_id = owners.id
+WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
```
Comment (Gambit142) [inline queries.sql:72]: - [x] One of the requirements is that you include the snapshot of queries in your pull request summary. The snapshot for the result of this query is incorrect. Kindly include the correct snapshot. 👍

File: queries.sql [hunk 6]
```diff
@@ -49,4 +49,33 @@ GROUP BY species;
...
+
+SELECT animals.name FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE escape_attempts = 0 AND owners.full_name = 'Dean Winchester';
+
```
Comment (Gambit142) [inline queries.sql:77]: - [x] One of the requirements is that you include the snapshot of queries in your pull request summary. The snapshot for the result of this query is incorrect. Kindly include the correct snapshot. 👍

File: queries.sql [hunk 7]
```diff
@@ -49,4 +49,33 @@ GROUP BY species;
...
+SELECT owners.full_name, COUNT(animals.id) as count FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id
+GROUP BY owners.id ORDER BY count DESC
+LIMIT 1;
```
Comment (Gambit142) [inline queries.sql]: - [x] The query gives the wrong result. The correct query should give the name and number of animals of the person that owns the most animals. 👍 

Hint: When creating the `animals` table, that `ID` was not serial, so your current `animals` table has no data for ID. So, I would advise you to either make the `ID` serial or change `count(animals.id)` to `count(animals.species_id)`. 👍 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @JacintaNgwu 👋 , 

Good job so far!

## Highlights
- Everything that creates or changes the structure of the tables is in the _**schema.sql**_ file
- Everything that populates tables with data is in the _**data.sql**_ file
- Everything that is a query that helps to analyze data is in the _**queries.sql**_ file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @JacintaNgwu 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.
- Your linters are passing all the checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**